### ISSUE TITLE
Please describe using taints and tolerations at Readme.md

### ISSUE THREAD

**Issue by WAlekseev** [kube-hetzner/terraform-hcloud-kube-hetzner#418]
Greets mysticaltech!

Thanks for sharing solution.

I have a troubles with taints under _nodepools at cluster generation stage: when I define some taint policy (in my case cilium enabled) I can't generate cluster, terraform failed to create cluster.

**Comment by mysticaltech** [comment]
@WAlekseev Yes, taints will block the execution of any pods so you cannot even complete the setup! Your solution here, if you really need those taints, is to create another nodepool, without taints, let it run.

Also, you can also set `allow_scheduling_on_control_plane` to true, and it will have the same effect of completing the installation.

However, please note that from an outside perspective at least, it does not make sense to have all agent nodes tainted (to me at least).

**Comment by mysticaltech** [comment]
Am converting this to a discussion, where it fits more!
